firebase-firestore/ktx/src/main/kotlin/com/google/firebase/firestore/ktx/Firestore.kt

```diff
@@ -162,3 +171,49 @@ class FirebaseFirestoreKtxRegistrar : ComponentRegistrar {
     override fun getComponents(): List<Component<*>> =
             listOf(LibraryVersionComponent.create(LIBRARY_NAME, BuildConfig.VERSION_NAME))
 }
+
+/**
+ * Starts listening to the document referenced by this `DocumentReference` with the given options and emits its values via a [Flow].
+ *
+ * - When the returned flow starts being collected, an [EventListener] will be attached.
+ * - When the flow completes, the listener will be removed.
+ *
+ * @param metadataChanges controls metadata-only changes. Default: [MetadataChanges.EXCLUDE]
+ */
+fun DocumentReference.snapshots(
+    metadataChanges: MetadataChanges = MetadataChanges.EXCLUDE,
```

Comment:
```suggestion
    metadataChanges: MetadataChanges = MetadataChanges.EXCLUDE
```

Reply:
Yup, sorry, just realized that. It's fixed in https://github.com/firebase/firebase-android-sdk/pull/1252/commits/692114cc815a109667461a817d501ed7872bd3d5.

Reply:
Also interestingly this seemed to be a lint-only issue as Kotlin 1.4 is perfectly happy with the trailing coma

Reply:
yeah, I think our version of ktlint is pretty ancient, we are planning to upgrade it eventually

Reply:
@vkryachko I think all comments have been addressed. Can you please approve the pending CI workflows and review the PR once again?